addon/mixins/node-actions.js

```diff
@@ -0,0 +1,100 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+    node: null,
+    model: null,
+    _node: Ember.computed.or('node', 'model'),
```

Comment:
Since this is mixin class, it's expected to be combined with a route, controller, or component. For:
- a route: falls back to model, or uses `Ember.computed.alias` to point to another attribute, e.g.:
```javascript
model() {
    node: Ember.computed.alias('model.node'),
    return new Ember.RSVP.hash({
        node: ...,
        user: ...
    })
}
```
- a controller: node could be set by `setupController` in the route or could fall back to model
- a component: node is passed down from the parent component/controller